aem/aem_skyline_install.md

```diff
@@ -3,80 +3,33 @@
-* The keystore added into the AEM `eventproxy-service` user&rsquo;s keystores vault
-* An Adobe I/O Console project and workspace is created using the public certificate
+3. [prepare the Adobe I/O Events service user keystore](aem_keystore_setup.md) 
+4. [set up your workspace in Adobe I/O Developer console, and as OSGI configuration](aem_console_setup.md)
 
-To execute the final steps for the Adobe IMS configuration needed by Adobe I/O Events, follow instructions below:
 
-1. Open your AEM `Tools` menu, or select the `Security` section, and then select `Adobe IMS Configurations` tile.
-
-   ![Adobe IMS Configurations UI](../img/events_aem_adobe-ims-conf-1.png "Adobe IMS Configurations UI")
-
-2. Click **Create**; 
-**Select `AdobeIOEvents`** in the Cloud Solution drop down; once selected, you should see the certificate you just added to the AEM `eventproxy-service` user's keystore;
-   Click **Next**
-
-    ![Adobe IMS Configuration Creation](../img/events_aem_adobe-ims-conf-2.png "Adobe IMS Configuration Creation")
-
-3. Fill in the various entries expected to configure the IMS account associated with the integration
- you just [created in the Adobe I/O Console](aem_console_setup.md):
-     * For **Title**: specify **Adobe IO Events** (or any other title that makes sense to you).
-     * For **Authorization Server**: it should be `https://ims-na1.adobelogin.com` (unless the URL shown in the **JWT** tab of your integration page in the Adobe I/O Console is different),
-     * For **API key**: Provide the API key available in the `Credentials details` tab of your credentials `Service Account (JWT)` page in your Adobe I/O Console workspace
-     * For **Client Secret**: Provide the Client Secret available in the `Credentials details` tab of your credentials `Service Account (JWT)` page in your Adobe I/O Console workspace
-     * For **Payload**: Provide the JWT payload available in the `Generate JWT` tab of your credentials `Service Account (JWT)` page in your Adobe I/O Console workspace
-  
-   ![Adobe I/O Console Generate JWT tab](../img/console_generate_jwt_tab.png "Adobe I/O Console Generate JWT tab")
-
-4. Click **Create**
-
-    ![Adobe IMS Configuration IMS account form](../img/events_aem_adobe-ims-conf-3.png "Adobe IMS Configuration IMS account form")
-
-5. Now you should see this new Adobe I/O Events IMS Configuration, and you can select it to check its health.
-
-     ![Adobe IMS Configuration IMS Health Check](../img/events_aem_adobe-ims-conf-4.png "Adobe IMS Configuration Health Check")
-      
+3. [Deploy the code](https://docs.adobe.com/content/help/en/experience-manager-cloud-manager/using/how-to-use/deploying-code.html) 
+3. [Finalize the Adobe IMS configuration in AEM](aem_ims_config.md)
+   
+### Verify your AEM Cloud Service is showing in the Adobe I/O Developer console
+   
+Once the previous steps are completed, a new `AEM as a Cloud Service` `Events Providers` should appear in the Adobe I/O console. 
```

Comment:
Once the previous steps are completed, a new `AEM as a Cloud Service` provider should appear in the  `Events Providers` list in the Adobe I/O developer console.